chapters/en/Unit 1 - Fundamentals/imaging.mdx

```diff
@@ -0,0 +1,29 @@
+
+7. Spatial Filtering Techniques and Image Enhancement
+Spatial filtering encompasses a broad range of applications in image processing, primarily focused on modifying images by altering each pixel's value based on its neighbouring pixels' values. Techniques include linear spatial filters, which can either blur (lowpass filters) or sharpen (high pass filters) an image. The properties and applications of different filter kernels, such as the Gaussian filter and box filters, are contrasted. Sharpening filters, which emphasize transitions in intensity, are often implemented through digital differentiation techniques like the Laplacian, highlighting edges and discontinuities in an image.
+
+8. Image Restoration and Reconstruction Techniques
+Image restoration focuses on objectively recovering a degraded image using knowledge about the degradation phenomenon. It often involves modelling the degradation process and applying inverse processes to regain the original image. In contrast, image enhancement is more subjective, aiming to improve the visual appearance of an image. Restoration techniques include dealing with issues like noise, which can originate from various sources during image acquisition or transmission. Advanced filters, both adaptive and non-adaptive, are explored for their noise-reduction capabilities. In the field of medical imaging, particularly in computed tomography (CT), image reconstruction from projections is a crucial application.
+
+9. Colour in Image Processing
+Colour is a powerful descriptor in image processing, playing a crucial role in object identification and recognition. Colour image processing includes both pseudo-Colour and full-Colour processing. Pseudo-Colour processing assigns Colours to grayscale intensities, while full-Colour processing uses actual Colour data from sensors. Understanding the fundamentals of Colour, including human Colour perception, the Colour spectrum, and the attributes of chromatic light, is key. Different Colour models, such as RGB for monitors and cameras and CMY/CMYK for printing, standardize Colour representation in digital imaging.
+In the RGB Colour model, images comprise three component (i.e., channel) images, each for red, green, and blue. The pixel depth in RGB images determines the number of possible colours, with a typical full-colour image having a 24-bit depth (8 bits for each colour component). This allows for over 16 million possible colours. The RGB colour cube represents the range of colours achievable in this model, with the grayscale extending from black to white. The evolution of colour image processing reflects not only technological advancements but also a deepening understanding of colour science and its application in digital imaging.
+

```

Comment:
One things that is missing in the chapter is "File Compression, Image formats, Metadata". I think it is important to highlight lossless and loss compression. Image formats that are not jpg or png (i.e. tiff, zarr), metadata (what is metadata and why does it matter for imaging?)